src/library-authoring/data/apiHooks.ts

```diff
@@ -136,7 +145,7 @@ export const useUpdateLibraryMetadata = () => {
 
       const newLibraryData = {
         ...previousLibraryData,
-        title: data.title,
+        ...data,
```

Comment:
I had to make this change to avoid a runtime error when [toggling the "Allow Public Read" switch](https://github.com/openedx/frontend-app-authoring/pull/1362/files#diff-dc9b4d64552ce3bd823235ba2dbf6213d89f0aaa06c7c093372b7c8a1ee99fe9R111-R113): "Helmet expects a string as a child of <title>"

This happened because the library.title is used in the page `<title>` component, and because I'm only updating `allow_public_read`, there's no `data.title` available while the query cache is being updated.